style.css

```diff
@@ -5,189 +5,205 @@
 }
 /*CSS RESET*/
 
+@font-face {
+  font-family: "AVGARDD";
+  src: url(/fonts/AVGARDD.TTF);
+}
+@font-face {
+  font-family: "AVGARDN";
+  src: url(/fonts/AVGARDN.TTF);
+}
+@font-face {
+  font-family: "impact_imp";
+  src: url(/fonts/impact.ttf);
+}
+body{
+  display:flex;
+  flex-direction: column;
+  background-color: #efefef
+}
 .header {
   height: 285px;
-  width: 100%;
   background-image: url(images/header-bg.jpg);
 }
-.header-content {
+.header__content {
   display: flex;
   justify-content: center;
   align-items: end;
 }
-.header-content span {
-  display: inline-block;
-}
 .logo {
+  position:relative;
+  left:-6px;
   background-image: url(images/logo.gif);
-  width: 372px;
+  background-repeat: no-repeat;
+  width:50%;
   height: 91px;
-  margin-left: 30px;
 }
-.pagetitle {
+.page__title {
   background-image: url(images/header-img1.jpg);
-  width: 311px;
+  background-repeat: no-repeat;
+  background-position: 90%;
+  width:50%;
   height: 215px;
 }
-.menubar {
+.menu__bar{
   display: flex;
-  height: 32px;
-  justify-content: center;
-  align-items: end;
-  position: absolute;
-  top: 215px;
-}
-.menubar ul {
-  list-style-type: none;
 }
-.menubar li {
-  float: left;
+.menu__items{
+  width:50%;
 }
 
-.btndivider {
-  vertical-align: middle;
-  background-image: url(images/menu-devider.gif);
-  background-repeat: no-repeat;
-  display: inline-block;
-  width: 2px;
-  height: 32px;
-}
-.hidden {
-  background-image: none;
-}
-.menubar a {
+.menu__items ul{
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width:360px;
+  height:32px;
+  white-space: nowrap;
+}
+.menu__items li{
+  height:32px;
+}
+.menu__items a{
+  display:inline-block;
+  line-height:32px;
+  text-align: center;
+  width:100%;
   font-family: "Trebuchet MS", sans-serif;
   font-size: 10pt;
   font-weight: bold;
   text-decoration: none;
   color: white;
-  margin: 0 15px 0 15px;
 }
-.menubar li:first-child a:first-of-type {
-  margin: 0 15px 0 8px;
+
+.menu__divider {
+  background-image: url(images/menu-devider.gif);
+  background-repeat: no-repeat;
+  display: inline-block;
+  width: 2px;
+  height: 32px;
 }
-.menubar li:hover {
+
+.menu__bar a:hover {
   background-image: url(images/menu-hover.gif);
   background-repeat: no-repeat;
   background-position: top;
 }
-.mainbody {
-  width: 100%;
-  height: 800px;
-  background-color: rgb(239, 239, 239);
-  position: relative;
+.main__body {
+  display: flex;
+  justify-content: center;
+  height:700px;
```

Comment:
Iti spun asta acum, dar o sa ioti fie folositoare cand incepi sa muncesti in domeniu si trebuie sa faci un site `responsive` adica sa se adapteze in marime si pentru mobil. Deci nu e bine sa specifici width si height sau orice alt stil care seteaza marimea in pixel. E bine sa folosesti valori alternative gen: `70%` sau `70vw` (vw vine de la View Width, adica 70% din marimea vederii, adica tot ce se vede in browser). Ar fii bine sa utilizezi si valorile alternative. 

Oricum, ar fii bine sa faci site-ul asta sa arate bine si pe mobil. Uite cum arata acum
![image](https://user-images.githubusercontent.com/6403637/230871520-f6e37fbe-d4d0-4dcb-a88f-4ef04a0d733a.png)

Ar fii bine sa reorganizezi elementele in pagina sa fie afisate pe verticala in asa fel incat sa arate bine pe mobil. Te las pe tine sa decizi prioritatea elementelor, dar fii atent ca si asta este important. Ca sa poti aplica diferite stiluri la diferite marimi ale device-ului trebuie sa te folosesti de `media queries`. Citeste aici https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries